.github/ISSUE_TEMPLATE/internal-docs-request.yaml

```diff
@@ -0,0 +1,94 @@
+name: "Internal documentation request (Elastic employees)"
+description: Request a documentation change or enhancement.
+title: "[Request] "
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Hello! This form will create an issue that the Security docs team will triage and prioritize. Please do not add any labels to your issue — we'll take care of that.
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: Describe what needs to be documented. What details do users need to know about? What are typical workflows for the feature?
+      placeholder: |
+        What: We're introducing new feature A.
+        Why: This feature will make X, Y, and Z easier for the user.
+        How: The user navigates to *Foo* → *Bar*, then clicks the *Wow* button.
+    validations:
+      required: true
```

Comment:
I agree, a text field is more flexible for additional info. Minor edit to suggestion above: `label: API docs impact`

Reply:
Actually, once I committed the suggestion and looked at the preview, it seems like maybe this should go lower in the order of fields, maybe between "Feature differences" and "Prerequisites...." IMO it kind of sticks out toward the top. It's a required field so the submitter can't overlook it even if it's lower.

Reply:
Moved in https://github.com/elastic/security-docs/pull/4308/commits/5dc55c88ac51157206a3ff75ee016b1fea574566, we can revert or relocate elsewhere if needed

Reply:
Agree with having a text field rather than a checkbox so that more info can be provided. 

I can't seem to add a suggestion for line 24. I'd maybe update the description to something slightly more specific, like:
Provide endpoint and parameter descriptions, and requests and response examples.

Reply:
I think you can't comment because the suggestion might be outdated? I'll make the change directly...